tests/network/sriov.go

```diff
@@ -689,6 +707,33 @@ func waitVMI(vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error)
 	return virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &k8smetav1.GetOptions{})
 }
 
+// deleteVMI deletes the specified VMI and waits for its absence.
+// Waiting for the VMI removal is placed intentionally for VMI/s with SR-IOV networks in order
+// to assure resources (VF/s) are fully released before reused again on a new VMI.
+func deleteVMI(vmi *v1.VirtualMachineInstance) error {
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		panic(err)
+	}
+
+	GinkgoWriter.Println("sriov:", vmi.Name, "deletion started")
```

Comment:
@EdDev WDYT about adding `vmi.Namespace` to this log entry as well?

Reply:
The name is unique and there is only one ns for the sriov tests. I think it is refundant.

Reply:
Thanks.